addon/components/table-columns.js

```diff
@@ -133,12 +139,13 @@ export default Ember.Component.extend({
       }
 
       // TODO make this smarter by taking option if we should do this
-      let shouldFetch = isEmpty(rowGroup.data) && !rowGroup.isCollapsed;
+      let rowData = rowGroup.get(this.get('rowGroupDataName'));
```

Comment:
this should be `get(rowGroup, this.get('rowGroupDataName'))` since `rowGroup` is usually a POJO.